newrelic/resource_newrelic_cloud_aws_integrations.go

```diff
@@ -579,6 +580,10 @@ func resourceNewRelicCloudAwsIntegrationsRead(ctx context.Context, d *schema.Res
 
 	linkedAccount, err := client.Cloud.GetLinkedAccountWithContext(ctx, accountID, linkedAccountID)
 	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
```

Comment:
Can you check if 
```
if _, ok := err.(*errors.NotFound); ok {
	d.SetId("")
	return nil
}
```
would work? This should be the default way to check if something doesn't exist anymore.

Reply:
I tried using it for aws and it didn't work. To add more context `"message": "Linked account 'xxxxx' for account 'xxxxx' not found."` is the err message returned from the API.